tests/Unit/DateTimeTest.php

```diff
@@ -3,31 +3,30 @@
 namespace Tests\Unit;
 
 use Carbon\Carbon;
+use DateTime;
 use Tests\TestCase;
-use Illuminate\Foundation\Testing\WithFaker;
-use Illuminate\Foundation\Testing\RefreshDatabase;
 
 class DateTimeTest extends TestCase
```

Comment:
This whole test file looks to me like it's testing the `\DateTime` class, and nothing really related to Pixelfed. Is there a reason to keep these tests?